corehq/apps/reports/filters/controllers.py

```diff
@@ -118,6 +133,14 @@ def _get_users(self, query, start, size, include_inactive=False):
             accessible_location_ids = SQLLocation.active_objects.accessible_location_ids(
                 self.request.domain, self.request.couch_user)
             users = users.location(accessible_location_ids)
+        if IS_ICDS_ENVIRONMENT and self.request.couch_user.get_role(self.domain).name == CPMU_ROLE_NAME:
```

Comment:
Did we consider doing this for all non domain admins/superusers?

Reply:
Ohh! Sorry. It skipped my mind. 😅 
I will ping on the ticket.